fdbserver/workloads/StorageServerCheckpointRestoreTest.actor.cpp

```diff
@@ -123,10 +123,7 @@ struct SSCheckpointRestoreWorkload : TestWorkload {
 			}
 		}
```

Comment:
This block is also present in `checkpointRestore()`. Can be refactored as an actor function.